opengemini/write.go

```diff
@@ -56,3 +56,46 @@ func (c *client) WriteBatchPoints(database string, bp *BatchPoints) error {
 	}
 	return nil
 }
+
+func (c *client) WritePoint(database string, point *Point, callbackFunc func(error)) error {
+	var buffer bytes.Buffer
+
+	var writer io.Writer
+
+	if c.config.GzipEnabled {
+		writer = gzip.NewWriter(&buffer)
+	} else {
+		writer = &buffer
+	}
+
+	if _, err := io.WriteString(writer, point.String()); err != nil {
```

Comment:
should we check if the return length exactly equals point serialize length?

Reply:
io.Writer and gzip.Writer both operate bytes.Buffer ,  WriteString appends the contents of s to the buffer, growing the buffer as needed. The return value n is the length of s; err is always nil. If the buffer becomes too large, WriteString will panic with ErrTooLarge.